resources/assets/js/annotations/annotatorContainer.vue

```diff
@@ -440,35 +427,43 @@ export default {
             if (this.isEditor) {
                 let promise;
                 // LabelBOT
-                if (!this.selectedLabel && this.labelbotIsOn) {
+                if (!this.selectedLabel && this.isLabelbotOn) {
+
+                    this.labelbotState = LABELBOT_STATES.COMPUTING;
 
-                    this.labelbotState = 'computing';
-                    
                     promise = this.generateFeatureVector(annotation.points)
-                    .then((featureVector) => {
-                        // Assign feature vector to the annotation
-                        annotation.feature_vector = featureVector;
-                    })
-                    .catch(handleErrorResponse);
+                        .then((featureVector) => {
+                            // Assign feature vector to the annotation
+                            annotation.feature_vector = featureVector;
+                        })
+                        .catch(handleErrorResponse);
                 } else {
-                    promise = Promise.resolve()
+                    promise = Promise.resolve();
                     annotation.label_id = this.selectedLabel.id;
                 }
                 // TODO: confidence control
                 annotation.confidence = 1;
-                promise.then(() => {
-                    return AnnotationsStore.create(this.imageId, annotation)
-                    .then((createdAnnotation) => {
-                        if (this.labelbotIsOn) {
-                            this.showLabelbotPopup(createdAnnotation);
+
+                promise
+                    .then(() => {
+                        return AnnotationsStore.create(this.imageId, annotation)
+                            .then((createdAnnotation) => {
+                                if (this.isLabelbotOn) {
+                                    this.showLabelbotPopup(createdAnnotation);
+                                }
+                                return createdAnnotation;
+                            })
+                            .then(this.setLastCreatedAnnotation);
+                    })
+                    .catch((e) => {
+                        if (this.isLabelbotOn) {
+                            this.updateLabelbotState(LABELBOT_STATES.OFF);
+                            Messages.danger("There are no annotations associated with any labels in this project!");
```

Comment:
This might show a misleading message if something else went wrong while LabelBOT was active. You should only show this if LabelBOT did not return any suggestions. If no annotations are present in the project, users should not be able to activate LabelBOT in the first place.